renovate.json

```diff
@@ -519,7 +519,7 @@
       ],
       "labels": [
         "release_note:skip",
-        "backport:skip",
+        "backport:prev-minor",
```

Comment:
__question:__ Sorry, I always forget what our labels mean 🙂. If we merge something into the `main` branch now, which is `9.0`, is `prev-minor` essentially equivalent to `backport:skip` since there is no prev-minor for 9.0?

Reply:
It's confusing, but AFAIK, `prev-minor` ships it to `8.x` at the moment.

The labels' descriptions explain the targets to each one of them ([source](https://github.com/elastic/kibana/labels?q=backport%3A)):
<img width="688" alt="image" src="https://github.com/user-attachments/assets/2f80059f-6997-47af-bf56-9da85e0e298a" />

Reply:
Ah, I see, nice, thanks!